app/src/main/res/layout/list_item_plant.xml

```diff
@@ -51,14 +51,14 @@
             android:layout_width="0dp"
             android:layout_height="wrap_content"
             android:layout_marginBottom="@dimen/margin_normal"
-            android:layout_marginEnd="@dimen/margin_normal"
-            android:layout_marginStart="@dimen/margin_normal"
+            android:layout_marginEnd="@dimen/margin_small"
+            android:layout_marginStart="@dimen/margin_small"
             android:layout_marginTop="@dimen/margin_small"
             android:text="@{vm.content}"
             android:textAppearance="?attr/textAppearanceListItem"
             app:layout_constraintEnd_toEndOf="parent"
             app:layout_constraintStart_toStartOf="parent"
-            app:layout_constraintTop_toBottomOf="@id/plant_item_image"
+            app:layout_constraintTop_toBottomOf="@+id/plant_item_image"
```

Comment:
+id isn't required here

Reply:
i know it doesnt, but adding it 'fixes' the preview designer to show it below the image view

Reply:
I'm unable to reproduce this with Android Studio 3.2 Canary 18.  Try Invalidating Caches / Restart from the Android Studio File menu and see if that fixes things.

Adding a + also introduces a bug by adding itself as an option for "Go to declaration".  I think we should remove the +.

Reply:
added another commit to remove it